docker-entrypoint.sh

```diff
@@ -80,7 +80,16 @@ if expr "$*" : '^java .*/start\.jar.*$' >/dev/null ; then
 		set -- $(cat /jetty-start)
 	else
 		# Do a jetty dry run to set the final command
-		set -- $("$@" --dry-run | sed 's/\\$//' )
+		"$@" --dry-run > /jetty-start
+		EXIT=$?
+		if [ $(egrep -v '\\$' /jetty-start | wc -l ) -gt 1 ] ; then
```

Comment:
What's the intent of this `'\\$'`? It seems like the `awk` invocation below is assuming it's the first character, so should it be anchored with `^`?